tasks/test.py

```diff
@@ -1168,3 +1170,115 @@ def get_modified_files(ctx):
+    if os.path.isfile("test_output_fast.json"):
+        failed_tests_fast, n_test_fast = parse_test_log("test_output_fast.json")
+        fast_success = len(failed_tests_fast) == 0
+    else:
+        print("test_output_fast.json not found, assuming no tests were run")
+
+    timestamp = int(datetime.now().timestamp())
+    print("Sending unit tests stats to Datadog")
+
+    print(f"Classic test executed: {n_test_classic}")
+    series.append(
+        create_count(
+            "datadog.ci.unit_tests.executed",
+            timestamp,
+            n_test_classic,
+            tags=[
+                "experimentation:fast-tests",
+                "test_type:classic",
+                "repository:datadog-agent",
+                f"pipeline_id:{os.getenv('CI_PIPELINE_ID')}",
+                f"job_name:{job_name}",
+            ],
+        )
+    )
+
+    print(f"Fast test executed: {n_test_fast}")
+    series.append(
+        create_count(
+            "datadog.ci.unit_tests.executed",
+            timestamp,
+            n_test_fast,
+            tags=[
+                "experimentation:fast-tests",
+                "test_type:fast",
+                "repository:datadog-agent",
+                f"pipeline_id:{os.getenv('CI_PIPELINE_ID')}",
+                f"job_name:{job_name}-fast",
+            ],
+        )
+    )
+
+    print(f"Classic test success: {classic_success}")
+    print(f"Fast test success: {fast_success}")
+
+    if fast_success == classic_success:
+        false_positive = 0
+        false_negative = 0
+    elif fast_success:
+        false_positive = 1
+        false_negative = 0
+    else:
+        false_positive = 0
+        false_negative = 1
+
+    series.append(
+        create_count(
+            "datadog.ci.unit_tests.false_positive",
+            timestamp,
+            false_positive,
+            tags=[
+                "experimentation:fast-tests",
+                "repository:datadog-agent",
+                f"pipeline_id:{os.getenv('CI_PIPELINE_ID')}",
+                f"job_name:{job_name}",
+            ],
+        )
+    )
+    series.append(
+        create_count(
+            "datadog.ci.unit_tests.false_negative",
+            timestamp,
+            false_negative,
+            tags=[
+                "experimentation:fast-tests",
+                "repository:datadog-agent",
+                f"pipeline_id:{os.getenv('CI_PIPELINE_ID')}",
+                f"job_name:{job_name}",
+            ],
+        )
+    )
+
+    send_metrics(series)
+
+
+def parse_test_log(log_file):
+    failed_tests = []
+    n_test_executed = 0
+    with open(log_file, "r") as f:
+        for line in f:
+            json_line = json.loads(line)
+            if json_line["Action"] == "fail" and "Test" in json_line:
+                n_test_executed += 1
+                failed_tests.append(json_line["Package"] + "/" + json_line["Test"])
```

Comment:
```suggestion
                failed_tests.append(os.path.join(json_line["Package"], json_line["Test"]))
```
and same below, I guess using this or `pathlib` is safer

Reply:
Hmmm, here I am not really interested about the path but the complete test name. I think I could use `pathlib` but not sure it is very useful

Reply:
ok so you could then maybe write `f"{json_line['Package']}/{json_line['Test']}"`, and this is a nitpick

Reply:
Yep that's one is better I think. Updating it